doc/geoipupdate.md

```diff
@@ -0,0 +1,87 @@
+# GEOIPUPDATE
+
+## NAME
+
+geoipupdate - GeoIP2, GeoLite2, and GeoIP Legacy Update Program
+
+## SYNOPSIS
+
+geoipupdate \[-Vvh\] \[-f license\_file\] \[-d target\_directory\]
+
+## DESCRIPTION
+
+`geoipupdate` automatically updates GeoIP2, GeoLite2, and GeoIP Legacy
+databases. The program connects to the MaxMind GeoIP Update server to
+check for new databases. If a new database is available, the program will
+download and install it.
+
+If you are using a firewall, you must have the DNS and HTTPS ports
+open.
+
+## OPTIONS
+
+* `-d` - Install databases to a custom directory. This must be specified
+  if `DatabaseDirectory` is not set in the configuration file.
+* `-f` - The configuration file to use. See [GeoIP.conf](GeoIP.conf.md)
+  for more information. *Required*.
+* `-h` - Display help and exit.
+* `-stack-trace` - Show a stack trace on any error message. This is
+  primarily useful for debugging.
+* `-V` - Display version information and exit.
+* `-v` - Enable verbose mode. Prints out the steps that `geoipupdate`
+  takes.
+
+## USAGE
+
+Typically you should run `geoipupdate` weekly. One way to achieve this
+is to use cron. Below is a sample crontab file that runs `geoipupdate`
```

Comment:
We could qualify this by saying something like if they are on a Linux like system they could use cron (thinking about other platforms).